src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json

```diff
@@ -0,0 +1,45 @@
+{
+  "ocr_data_fields":[
+    {
+      "name":"applicant1_firstName",
+      "value":"applicant1_firstName"
+    },
+    {
+      "name":"applicant1_lastName",
+      "value":"applicant1_lastName"
+    },
+    {
+      "name":"applicant1_telephoneNumber",
+      "value":"09999999990"
+    },
+    {
+      "name":"applicant1_address",
+      "value":"123 test street, London"
+    },
+    {
+      "name": "applicant1_relationToChild",
+      "value": "Step Father"
+    },
```

Comment:
pls check below tag is required:
 {
      "name": "applicant1_relationToChild",
      "value": "Step Father"
    },

Reply:
Testing optional field on A60